
<launch> 
  <node pkg="planning_environment" type="display_planner_collision_model" respawn="false" output="screen">
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />
    <param name="skip_collision_map" type="bool" value="true" />
  </node>
</launch>
